modules/installation-bare-metal-config-yaml.adoc

```diff
@@ -57,7 +57,8 @@ If you disable simultaneous multithreading, ensure that your capacity planning
 accounts for the dramatically decreased machine performance.
 ====
 <4> This parameter controls the number of workers that the cluster deploys for
-you, so you must set the number of workers to `0` in this file. Remember that you
+you. Because the cluster cannot provision workers for you,
+you must set the number of workers to `0` in this file. Remember that you
```

Comment:
I still find this wording confusing. I understand the cluster doesn't provision the hardware but it still deploys OCP to any worker nodes you boot with the worker ignite file.